integration-tests/aqua/examples/abilities.aqua

```diff
@@ -77,11 +77,13 @@ ability CCCC:
     arrow(x: i8) -> bool
     simple: SSS
 
-func checkAbCalls() -> bool, bool:
+func checkAbCalls() -> bool, bool, bool:
     closure = (x: i8) -> bool:
         <- x > 20
 
     MySSS = SSS(arrow = closure)
     MyCCCC = CCCC(simple = MySSS, arrow = MySSS.arrow)
+    res1 <- MySSS.arrow(42)
```

Comment:
Does this test the bug fix? 
Let's create a separate test with:
- Abilities returned from `func`
- Abilities with names `ab`, `Ab`, `AB` (or similar in terms of case)